weblab/templates/emails/user_created.txt

```diff
@@ -0,0 +1,5 @@
+A new WebLab user was created.
```

Comment:
Is it possible to add in the user_created email a link to the admin page where an email can give them additional permissions? Because that's the most common thing we'll want to do when someone registers - allow them to upload models if we know them!